### PR TITLE
Fix NPE with old ES version

### PR DESCRIPTION
**Pull request by MrCitron**
- [ X ] I have signed the [Contributor License Agreement (CLA)][]

[Contributor License Agreement (CLA)]: https://www.elastic.co/contributor-agreement/

Hi

A simple bug fix to make the library still working with older ES versions that do not provide cluster UUID info

### CODE REVIEW

**Comment by eric-poitras** [comment]
We have the same issue here.

**Comment by MrCitron** [comment]
Hi,

nobody available to review this ?

Should I backport the modification in 7.x branch or in 6.8 ?

Thank you

File: mr/src/test/java/org/elasticsearch/hadoop/rest/RestClientTest.java [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
 import org.elasticsearch.hadoop.util.BytesArray;
+import org.elasticsearch.hadoop.util.ClusterInfo;
 import org.elasticsearch.hadoop.util.FastByteArrayInputStream;
 import org.elasticsearch.hadoop.util.TestSettings;
 import org.junit.Test;
 import org.mockito.Mockito;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```
Comment (jbaiera) [inline mr/src/test/java/org/elasticsearch/hadoop/rest/RestClientTest.java]: Nit: Can you convert this into explicit imports?

Comment (MrCitron) [reply mr/src/test/java/org/elasticsearch/hadoop/rest/RestClientTest.java]: Ok sure

**Comment by MrCitron** [comment]
@jbaiera It seems like I can not change the source branch of this PR
I will close this one and open a new one if you don't mind ?

**Comment by MrCitron** [comment]
To be replaced by #1304

**Comment by jbaiera** [comment]
Replaced by #1304

**Pull request closed without merging**